contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -179,21 +150,13 @@ contract RaidenMicroTransferChannels {
         constant
         returns (address)
     {
-        // Create message which should be signed by sender
-        string memory message = getBalanceMessage(_receiver_address, _open_block_number, _balance);
-        uint message_length = bytes(message).length;
-        string memory message_length_string = uintToString(message_length);
-
-        // Prefix the message
-        string memory prefixed_message = concat(prefix, message_length_string);
-        prefixed_message = concat(prefixed_message, message);
-
-
-        // Hash the prefixed message string
-        bytes32 prefixed_message_hash = keccak256(prefixed_message);
+        var message_hash = keccak256(
```

Comment:
you and me both know that this should be kept in sync with the function, but just add a comment here for future maintainers that these hashed strings should be in sync with this function's parameters